YoCode/DuplicationCheck.cs

```diff
@@ -55,28 +55,36 @@ public DuplicationCheck(PathManager dir, string CMDtoolsDirConfig, IFeatureRunne
         }
 
         private void ExecuteTheCheck() {
-            var origEvidence = RunAndGatherEvidence(origArguments);
-            var modEvidence = RunAndGatherEvidence(modiArguments);
+            (var origEvidence, var origCodeBaseCost, var origDuplicateCost) = RunAndGatherEvidence(origArguments,"Original");
+            (var modEvidence, var modCodeBaseCost, var modDuplicateCost) = RunAndGatherEvidence(modiArguments,"Modified");
 
             if (origEvidence.FeatureFailed || modEvidence.FeatureFailed)
             {
                 DuplicationEvidence.SetFailed($"Failed: Original={origEvidence.FeatureFailed}, Modified={modEvidence.FeatureFailed}");
                 return;
             }
 
+            OrigCodeBaseCost = origCodeBaseCost;
```

Comment:
Do these need to be properties? It looks like you could remove them?

Reply:
They are used as evidence